docs/dev/development/documentation.rst

```diff
@@ -0,0 +1,85 @@
+#############
+Documentation
+#############
+
+Developer documentation
+=======================
+
+The developer documentation is hosted at `warehouse.pypa.io`_. It's written in
+`reStructuredText`_ and built using `Sphinx`_.
```

Comment:
We allow Markdown as well, by including `myst_parser`. See https://github.com/pypi/warehouse/blob/51ffd8ff87d763509486ac3cd03e07cdb92be693/docs/dev/architecture.md for one example.